servant-client/test/Servant/ClientTestUtils.hs

```diff
@@ -145,6 +145,34 @@ instance ToDeepQuery Filter where
     , (["name"], Just (Text.pack name'))
     ]
 
+-----------------------------
+-- MultiVerb test endpoint --
+-----------------------------
+
+-- Possible errors
+
+type GetUserMultiVerbErrors =
+  '[RespondEmpty 404 "User not found"]
```

Comment:
since this is built into servant, maybe an example with another error, or some differently typed outcome that is not an error, would be better?

like, an end-point that receives a query param with a boolean that decides whether it should return an `Int` or a `Bool`?

Reply:
...  or an integer query param: if it is even, respond with an integer; if it's odd, respond with a boolean; if it's negative, respond with a 400 error?